pkg/util/scrubber/default.go

```diff
@@ -94,11 +94,17 @@ func AddDefaultReplacers(scrubber *Scrubber) {
 		Regex: regexp.MustCompile(`(?i)([a-z][a-z0-9+-.]+://|\b)([^:]+):([^\s|"]+)@`),
 		Repl:  []byte(`$1$2:********@`),
 	}
-	passwordReplacer := matchYAMLKeyPart(
+	yamlPasswordReplacer := matchYAMLKeyPart(
 		`(pass(word)?|pwd)`,
 		[]string{"pass", "pwd"},
 		[]byte(`$1 "********"`),
 	)
+	// replace all alphanum + special chars except quotes
+	passwordReplacer := Replacer{
+		// see test cases for match patterns
+		Regex: regexp.MustCompile(`(?i)(\"?(?:pass(?:word)?|pswd|pwd)\"?)( = |=|: \"?)([0-9A-Za-z#!$%&()*+,\-./:<=>?@[\\\]^_{|}~]+)`),
```

Comment:
Wondering if we could simply the regex a bit. 

`(?i)(\"?(?:pass(?:word)?|pswd|pwd)\"?)([:=\s]\"?)([^\s]+)`

Replaced `( = |=|: \"?)` with `([:=\s]\"?)` That would `=` `:`, space and the optional `"`

Replaced `([0-9A-Za-z#!$%&()*+,\-./:<=>?@[\\\]^_{|}~]+)` with `([^\s]+)` matches anything that is not a space

Reply:
We might as well use `[[:space:]]`, it includes all whitespace characters
https://pkg.go.dev/regexp/syntax
```
[[:space:]]    whitespace (== [\t\n\v\f\r ])
```